### DISCUSSION TITLE
Can I temporarily disable the scaled object from scaling?

### DISCUSSION THREAD

**Discussion by bkruger99** [kedacore/keda#1331] [Q&A / Need Help] [answered]
Hi,

I am curious if I can disable/enable a scaled object w/o deleting it?  Maybe adding an annotation or something along the lines w/o outright deleting it?  Specifically looking for something more CI/CD friendly for deployment, but want to limit exposure on the cluster of what someone needs for privileges, but may need to disable autoscaling temporarily to troubleshoot issues/set a fixed number of replicas/ etc.  A deployment scaled to zero for instance will cause the HPA to ignore and do nothing.  In the case of keda, if I scale deployment to zero, keda will scale it back up.

Thanks!

**Comment by zroubalik** [comment, marked as answer]
Hi, unfortunately this is currently not possible. But feel free to open an issue/proposal for this!

**Comment by wiseelf** [comment]
Has something changed?

**Reply by JorTurFer** [reply]
you could be interested on this: https://github.com/kedacore/keda/issues/944

**Comment by fty4** [comment]
It is possible with:

```yaml
metadata:
  annotations:
    autoscaling.keda.sh/paused: "true"
```

find docs [pausing autoscaling](https://keda.sh/docs/2.19/concepts/scaling-deployments/#pausing-autoscaling)
